### ISSUE TITLE
python -m fastchat.serve.vllm_worker --model-path Qwen/Qwen-14B-Chat-Int4 --trust-remote-code   vllm 支持量化模型吗？我跑Qwen-14B-Chat-Int4 报错[BUG]

### ISSUE THREAD

**Issue by micronetboy** [QwenLM/Qwen#651]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

python -m fastchat.serve.vllm_worker --model-path Qwen/Qwen-14B-Chat-Int4 --trust-remote-code   
vllm 支持量化模型吗？我跑Qwen-14B-Chat-Int4 报错

(qwenvllm) [venus@localhost llm]$ python -m fastchat.serve.vllm_worker --model-path Qwen/Qwen-14B-Chat-Int4 --trust-remote-code
[2023-11-17 07:57:48,956] [INFO] [real_accelerator.py:158:get_accelerator] Setting ds_accelerator to cuda (auto detect)
2023-11-17 07:57:52.313603: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-11-17 07:57:53.161573: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
/home/venus/anaconda3/envs/qwenvllm/lib/python3.10/site-packages/scipy/__init__.py:146: UserWarning: A NumPy version >=1.17.3 and <1.25.0 is required for this version of SciPy (detected version 1.25.0
  warnings.warn(f"A NumPy version >={np_minversion} and <{np_maxversion}"
INFO 11-17 07:57:55 llm_engine.py:72] Initializing an LLM engine with config: model='Qwen/Qwen-14B-Chat-Int4', tokenizer='Qwen/Qwen-14B-Chat-Int4', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=True, dtype=torch.float16, max_seq_len=8192, download_dir=None, load_format=auto, tensor_parallel_size=1, quantization=None, seed=0)
WARNING 11-17 07:57:57 tokenizer.py:66] Using a slow tokenizer. This might cause a significant slowdown. Consider using a fast tokenizer instead.
2023-11-17 07:57:57 | INFO | torch.distributed.distributed_c10d | Added key: store_based_barrier_key:1 to store for rank: 0
2023-11-17 07:57:57 | INFO | torch.distributed.distributed_c10d | Rank 0: Completed store-based barrier for key:store_based_barrier_key:1 with 1 nodes.
2023-11-17 07:57:57 | INFO | torch.distributed.distributed_c10d | Added key: store_based_barrier_key:2 to store for rank: 0
2023-11-17 07:57:57 | INFO | torch.distributed.distributed_c10d | Rank 0: Completed store-based barrier for key:store_based_barrier_key:2 with 1 nodes.
2023-11-17

### 期望行为 | Expected Behavior

正常运行

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by qiezikuangge** [comment]
vllm 不支持auto-gptq量化

**Comment by jklj077** [comment]
您好，vllm官方尚不支持AutoGPTQ的量化方案，建议关注<https://github.com/vllm-project/vllm/pull/916>和其简化版<https://github.com/vllm-project/vllm/pull/1580>两个PR的情况（现在看前面这个不会merge，后面这个有潜在的巨大性能问题）。
